spec/unit/repository_migrator_spec.rb

```diff
@@ -14,6 +14,17 @@
     end
   end
 
+  context "when forcing" do
+    before do
+      expect_any_instance_of(FedoraMigrate::RepositoryMigrator).to receive(:source_objects).and_return([])
```

Comment:
Are these tests or stubs?  I'm guessing you want `allow_any_instance_of` here instead of `expect_any_instance_of` for these two lines.